website/blog/2019-05-19-15-verdaccio-4-release.md

```diff
@@ -0,0 +1,219 @@
+---
+author: Ayush Sharma
+authorURL: https://twitter.com/ayusharma_
+authorFBID: 1122901551
+title: Verdaccio 4 released today !!!
+---
+
+#### Release name: Freedom
+
+Verdaccio is a free open source javascript package proxy registry. It is fully compatible with pnpm, yarn and npm package management clients. It follows the CommonJS compliant package specifications. 
+
+You can install and upgrade to the latest version by following commands:
+using **npm** 
+````
+npm install -g verdaccio@4.0.0
+````
+
+or using **Yarn** 
+````
+yarn global add verdaccio@4.0.0
+````
+
+or using **pnpm** 
+````
+yarn global add verdaccio@4.0.0
+````
+
+You can find detailed installation instructions [here](https://verdaccio.org/docs/en/installation)
+
+## Why 'Freedom' ?
+Verdaccio originated from [sinopia](https://github.com/rlidwka/sinopia) almost three years ago and since then verdaccio team maintaining and releasing major release every year. Since the fork, the project has evolved in many ways, making the project’s code base modern, easier to debug and more straightforward to contribute to by the community. 
```

Comment:
I think it is better mention Sinopia and Verdaccio capitalized, what do you think @ayusharma? (if the answer is truthy, there are several mentions to capitalize, so I can review again and mark them :wink:)
:thinking::thinking::thinking: should we mention also team capitalized (Verdaccio Team)?? cc: @juanpicado